fuzz/fuzz_targets/cranelift-fuzzgen.rs

```diff
@@ -146,6 +147,9 @@ pub struct TestCase {
     /// Functions under test
     /// By convention the first function is the main function.
     pub functions: Vec<Function>,
+    /// Control planes for function compilation.
+    /// There should be an equal amount as functions to compile.
+    pub ctrl_planes: Vec<ControlPlane>,
```

Comment:
I didn't combine these in a `Vec<(Function, ControlPlane)>`, because it makes the diff a little cleaner and in the manual arbitrary implementation it can still be controlled that the two vectors have the same size.